src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Channel.xml

```diff
@@ -40,12 +40,31 @@
         <itemOperations>
             <itemOperation name="admin_get">
                 <attribute name="method">GET</attribute>
-                <attribute name="groups">
-                    <attribute>channel:read</attribute>
+                <attribute name="path">/admin/channels/{id}</attribute>
+                <attribute name="openapi_context">
+                    <attribute name="summary">Use $code to retrieve a channel resource.</attribute>
+                </attribute>
+            </itemOperation>
+
+            <itemOperation name="shop_get">
+                <attribute name="method">GET</attribute>
+                <attribute name="path">/shop/channels/{id}</attribute>
+                <attribute name="normalization_context">
+                    <attribute name="groups">shop:channel:read</attribute>
+                </attribute>
+                <attribute name="openapi_context">
+                    <attribute name="summary">Use $code to retrieve a channel resource.</attribute>
                 </attribute>
             </itemOperation>
         </itemOperations>
 
+        <subresourceOperations>
+            <subresourceOperation name="shop_billing_data_get_subresource">
```

Comment:
```suggestion
            <subresourceOperation name="admin_billing_data_get_subresource">
```

Reply:
Or maybe path should be with /shop/ prefix ?

Reply:
`shopBillingData` is a name of an associated entity, so `shop` prefix is necessary and it is not a context prefix 🎉